pyproject.toml

```diff
@@ -5,10 +5,10 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "niftyone"
 description = "Large-scale neuroimaging visualization using FiftyOne"
-authors = [{ name = "Connor Lane", email = "connor.lane858@gmail.com" }]
+authors = [{name = "Connor Lane", email = "connor.lane858@gmail.com"}]
```

Comment:
Personal email?

Reply:
I left this as is from the original and never updated it. We could update this - I'll leave it up to @clane9.

Reply:
This is just the email I use for github. I don't see a strong reason to change. But I do think @kaitj you should add yourself as another author.

Reply:
Also in #16 I believe there was perhaps a ruff update that applied these nonsignificant formatting diffs. Can we rebase/ fix the ruff version so we don't undo these diffs?

Reply:
I'll take a look - you're right, there was likely an update in the formatting somewhere.

Reply:
Okay, just made a new commit that adds myself as another author. Re: formatting - I fixed it in the same commit, but its not a ruff issue. It's my vscode "formatOnSave" feature that is changing the formatting.

Reply:
I see, I usually try to avoid that to get less spam from bots crawling GitHub. But 🤷

Reply:
That's why I use this email. It's a trash public email that gets a lot of spam. What is your suggestion @nx10?

Reply:
Whatever you are comfortable with, I was planning on changing all of mine to dair@childmind.org soon. I don't think it matters outside of spam protection.